src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionString.cs

```diff
@@ -14,6 +14,9 @@ public class SqlConnectionString : ConnectionString
 
         protected override void ValidateImpl(ref List<string> errors)
         {
+            // Validate ConnectionString.FactoryName
+            SqlProviderParser.GetSupportedProvider(FactoryName);
```

Comment:
This won't add anything to `errors` but it will just throw. Is it what we want?

Reply:
fixed, now we add message to errors